sktime/networks/cnn.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""Time Convolutional Neural Network (CNN) (minus the final output layer)."""
+
+__author__ = ["James-Large, Withington, TonyBagnall"]
+
+from sktime.networks.base import BaseDeepNetwork
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+_check_soft_dependencies("tensorflow", severity="error")
```

Comment:
this should be a warning, and inside the `__init__` with an error.
See the "soft dependencies" docs.

Also, it should be `_check_dl_dependencies`.